make-release.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/&\n\n## $RELEASE_VERSION\n\n\
+% RELEASE NOTES TEMPLATE - Lines starting with '%' will be ignored.\n\
+% This description should answer \"who needs this update?\"\n\
+% Please enter your description right below, with no empty lines.\n/" CHANGES.md
```

Comment:
Please start your description in the empty line below, without adding new empty lines

Reply:
It was also necessary to update the offset passed to the editor

Reply:
I did the two fixes myself.

The resulting line was a bit longer than I'd like it; if you are comfortable with the size, feel free to merge this :)